packages/constants/src/enumRegistry.ts

```diff
@@ -169,13 +169,21 @@ export const prefixMap = new Map(
   ]),
 );
 
+/**
+ * Converts a string from formats like SNAKE_CASE to camelCase
+ * Keep in sync with packages/shared/src/utils/enumRegistry.js
+ * @param {string} value - The string to convert
+ * @returns {string} The converted string in camelCase
+ */
+const toCamelCase = (value: string): string => {
```

Comment:
It was a bit of a shame that I had to redefine this function again here as well as in utils but the constants package doesn't have utils installed as a dependency so I had to do it.